picture.go

```diff
@@ -13,10 +13,12 @@ package excelize
 
 import (
 	"bytes"
+	"compress/gzip"
 	"encoding/json"
 	"encoding/xml"
 	"image"
 	"io"
+	"io/ioutil"
```

Comment:
I suggest letting the user do the image format detection, and setting sheet background by specifying the format. In addition, the package `ioutil` has been deprecated in Go 1.16. We can use `io` or `os` instead of that.

Reply:
solved